website/content/api-docs/secret/databases/postgresql.mdx

```diff
@@ -51,6 +51,10 @@ has a number of parameters to further configure a connection.
 
 - `password` `(string: "")` - The root credential password used in the connection URL.
 
+- `self_managed` `(boolean: "false")` - <EnterpriseAlert product="vault" inline /> If
+  set, allows onboarding static roles with a rootless connection configuration. Mutually
+  exclusive with `username` and `password`.
```

Comment:
I can't remember exactly what the implementation does when we set `verify_connection=true`. Is it also mutually exclusive? If not should we say something here about that behavior? Something like "If true, will override verify_connection"?

Reply:
Good point, at present it is not mutually exclusive and `verify_connection` is overridden under the hood. Will add a note on the override behavior, thanks!